python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1300,3 +1319,27 @@ def test_iloc_with_lists(data, key):
     psr = pd.Series(data)
     gsr = cudf.Series(data)
     assert_eq(psr.iloc[key], gsr.iloc[key])
+
+
+@pytest.mark.parametrize("key", [5, -10, "0", "a"])
```

Comment:
I think for `bad_key` tests, there could be a test case for 0-d numpy array too? For example at this line, add `np.array(5)`, `np.array("a")`

Reply:
Done! Ditto below